src/runtime_src/core/common/device.cpp

```diff
@@ -78,6 +78,53 @@ register_axlf(const axlf* top)
 
   // Build modified CONNECTIVITY and MEM_TOPOLOGY section based on memory group ids
   // Base groups off data from driver
+  auto m_itr = m_axlf_sections.find(MEM_TOPOLOGY);
+  int g_count = (int)m_grp_info.size();
+  if (g_count && (m_itr != m_axlf_sections.end())) {
+    auto m_mem = reinterpret_cast<::mem_topology*>((*m_itr).second.data());
+    
+    /* Store the original memory topology before modify */
+    size_t size = (*m_itr).second.size();
+    char *buf = new char[size];
```

Comment:
No raw news.  use std::vector<char> buf(size);

Reply:
I will update it.